shell/src/main/java/alluxio/cli/ConfigurationDocGenerator.java

```diff
@@ -125,6 +125,83 @@ public static void writeCSVFile(Collection<? extends PropertyKey> defaultKeys, S
   }
 
   /**
+   * Writes description of property key to yml files.
+   *
+   * @param defaultKeys Collection which is from PropertyKey DEFAULT_KEYS_MAP.values()
+   * @param filePath    path for csv files
+   */
+  public static void writeYMLFile(Collection<? extends PropertyKey> defaultKeys, String filePath)
+      throws IOException {
+    if (defaultKeys.size() == 0) {
+      return;
+    }
+
+    FileWriter fileWriter;
+
+    //HashMap for yml file names
+    Map<String, String> fileNames = new HashMap<>();
```

Comment:
Tried to use one fileWriterMap but Jenkins reported errors:
1. alluxio.cli.ConfigurationDocGenerator.writeYMLFile(Collection, String) may fail to clean up java.io.Writer
2. Obligation to clean up resource created at ConfigurationDocGenerator.java:[line 62] is not dischargedPath continues at ConfigurationDocGenerator.java:[line 64]

Now, it had to use an array to store file names.